docs/_includes/common/icon-button.html

```diff
@@ -54,4 +54,48 @@ <h3 id="icon-button-badged">Badged Icon Button</h3>
 </button>
     {% endhighlight %}
 
+    <h3 id="icon-button-cart-nav"> Cart Nav Icon Button </h3>
```

Comment:
These two examples feel too specific. We just need a single example, and then clearly state that any icon can go on the grey matte background.